lab_3_keywords_textrank/main.py

```diff
@@ -278,7 +331,10 @@ def calculate_inout_score(self, vertex: int) -> int:
                 number of incidental vertices
         If vertex is not present in the graph, -1 is returned
         """
-        pass
+        if vertex in self._matrix[0]:
+            v_ind = self._matrix[0].index(vertex)+1
+            return self._matrix[v_ind][1:].count(1)
```

Comment:
is there a reason you do not use `sum`?

Reply:
i just didn't think of using that, i'll consider using sum

Reply:
done